llvm/include/llvm/TargetParser/X86TargetParser.def

```diff
@@ -248,10 +248,11 @@ X86_FEATURE_COMPAT(AVXVNNIINT16,    "avxvnniint16",           0)
 X86_FEATURE_COMPAT(SM3,             "sm3",                    0)
 X86_FEATURE_COMPAT(SHA512,          "sha512",                 0)
 X86_FEATURE_COMPAT(SM4,             "sm4",                    0)
-X86_FEATURE       (EGPR,            "egpr")
+X86_FEATURE_COMPAT(APXF,            "apxf",                   0)
 X86_FEATURE_COMPAT(USERMSR,         "usermsr",                0)
 X86_FEATURE_COMPAT(AVX10_1,         "avx10.1-256",            0)
 X86_FEATURE_COMPAT(AVX10_1_512,     "avx10.1-512",            0)
+X86_FEATURE       (EGPR,            "egpr")
```

Comment:
Could you put EGPR, CF together with other APX sub-features?

Reply:
We cannot change the position of X86_FEATURE_COMPAT at the whole list. So currently. we require some of APX strings to insert at those positions.

Reply:
I think only the position of X86_FEATURE_COMPAT matters while X86_FEATURE does not?

Reply:
E.g. we need 4 `X86_FEATURE` inserted between `X86_FEATURE_COMPAT(AVX512FP16,` and `X86_FEATURE_COMPAT(AVXIFMA,`